static/app/views/dashboards/manage/dashboardTable.tsx

```diff
@@ -0,0 +1,232 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import type {Location} from 'history';
+
+import {
+  createDashboard,
+  deleteDashboard,
+  fetchDashboard,
+} from 'sentry/actionCreators/dashboards';
+import {addErrorMessage, addSuccessMessage} from 'sentry/actionCreators/indicator';
+import type {Client} from 'sentry/api';
+import {ActivityAvatar} from 'sentry/components/activity/item/avatar';
+import {Button} from 'sentry/components/button';
+import {openConfirmModal} from 'sentry/components/confirm';
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
+import GridEditable, {
+  COL_WIDTH_UNDEFINED,
+  type GridColumnOrder,
+} from 'sentry/components/gridEditable';
+import Link from 'sentry/components/links/link';
+import TimeSince from 'sentry/components/timeSince';
+import {IconCopy, IconDelete} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types/organization';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import withApi from 'sentry/utils/withApi';
+import type {DashboardListItem} from 'sentry/views/dashboards/types';
+
+import {cloneDashboard} from '../utils';
+
+type Props = {
+  api: Client;
+  dashboards: DashboardListItem[] | undefined;
+  location: Location;
+  onDashboardsChange: () => void;
+  organization: Organization;
+  isLoading?: boolean;
+};
+
+enum Keys {
```

Comment:
```suggestion
enum ResponseKeys {
```

small nit, but just thinking that `Keys` is pretty generic, if we're going to make an enum we should make it clear what keys it's meant to be for